README.md

```diff
@@ -1,71 +1,152 @@
-- :x: : module version: `v1.1.0` and `v1.2.0` do not work on Kubernetes > `1.19.x` due to this [issue](https://github.com/open-policy-agent/gatekeeper/issues/820).
+| Kubernetes Version |   Compatibility    |                        Notes                        |
+| ------------------ | :----------------: | --------------------------------------------------- |
+| `1.20.x`           | :white_check_mark: | No known issues                                     |
+| `1.21.x`           | :white_check_mark: | No known issues                                     |
+| `1.22.x`           | :white_check_mark: | No known issues                                     |
+| `1.23.x`           |     :warning:      | Conformance tests passed. Not officially supported. |
+
+Check the [compatibility matrix][compatibility-matrix] for additional informations about previous releases of the modules.
+
+## Usage
+
+### Prerequisites
+
+|            Tool             |  Version  |                                                                          Description                                                                           |
+| --------------------------- | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [furyctl][furyctl-repo]     | `>=0.6.0` | The recommended tool to download and manage KFD modules and their packages. To learn more about `furyctl` read the [official documentation][furyctl-repo]. |
+| [kustomize][kustomize-repo] | `>=3.5.0` | Packages are customized using `kustomize`. To learn how to create your customization layer with `kustomize`, please refer to the [repository][kustomize-repo]. |
+| [KFD Monitoring Module][kfd-monitoring] | `>v1.10.0` | Expose metrics to Prometheus *(optional)* |
+
+> You can comment out the service monitor in the [kustomization.yaml][core-kustomization] file if you don't want to install the monitoring module.
+
+
+### Deployment
+
+1. List the packages you want to deploy and their version in a `Furyfile.yml`
+
+```yaml
+bases:
+  - name: opa/gatekeeper
+    version: "v1.6.0"
+```
+
+> See `furyctl` [documentation][furyctl-repo] for additional details about `Furyfile.yml` format.
+
+1. Execute `furyctl vendor -H` to download the packages
+
+1. Inspect the download packages under `./vendor/katalog/opa/gatekeeper`.
+
+1. Define a `kustomization.yaml` that includes the `./vendor/katalog/opa/gatekeeper` directory as resource.
+
+```yaml
+resources:
+- ./vendor/katalog/opa/gatekeeper
+```
+
+1. Apply the necessary patches. You can find a list of common customization [here](#common-customizations).
+
+1. To deploy the packages to your cluster, execute:
+
+```bash
+kustomize build . | kubectl apply -f -
+```
+
+### Common Customizations
+
+#### Disable constraints
+
+If you need to disable one of the constraints thats provided by default,
+you can do it by creating a kustomize patch like the following:
```

Comment:
I would make it more concise:

_"Disable one of the default constraints by creating the following kustomize patch: "_

Reply:
I like it!